operator/controllers/instance_controller.go

```diff
@@ -57,9 +66,9 @@ func (r *InstanceReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 	ctx, cancel := context.WithTimeout(context.Background(), instanceReconcileTimeout)
 	defer cancel()
 
-	log := r.Log.WithValues("instance", req.NamespacedName)
+	log := r.log.WithValues("instance", req.NamespacedName)
 
-	releaseReq := ReleaseRequest{req}
+	releaseReq := ReleaseRequest{req, r.metabrokerName}
```

Comment:
```suggestion
	releaseReq := ReleaseRequest{Request: req, prefix: r.metabrokerName}
```
When having more than one field, not naming them seems prone to issues as the code changes.

Reply:
Different types, but sure...

Reply:
Right; I didn't mean it for the computer, but for the dumb person (me) reading the code :)